dev/reference/tools/tidb-lightning/config.md

```diff
@@ -0,0 +1,324 @@
+---
+title: TiDB Lightning Configuration
+summary: CLI usage and sample configuration
+category: reference
+---
+
+# TiDB Lightning Configuration
+
+## Configuration files
+
+TiDB Lightning's global and task configurations have compatible structures.
+Unless [Server mode](/reference/tools/tidb-lightning/web.md) is enabled, the same file is used for both parts of configuration.
```

Comment:
Please combine this line with the previous line. Generally, do not manually break a line if you didn't mean to put them into two paragraphs.